### ISSUE TITLE
Ability to set download folder name

### ISSUE THREAD

**Issue by yacob841** [C9Glax/tranga-website#64]
I think it would be nice to be able to modify the download folder name (especially during initial adding to library). Some sites add weird tags in their names such as the group that is working on it or things like that, also I have one where it says x27 instead of ' . In these scenarios I would not like my reader to pickup these tags and issues so I would prefer to specify the download folder to not include them.

I have confirmed this is possible, I was able to modify a jobs file to a new folder name and it works.

**Comment by db-2001** [comment]
Yeah, will be addressed in one of the next few updates. The functionality exists on API side but not on website side. @C9Glax can you confirm that this feature is not the one that's been commented out in the code?

**Comment by C9Glax** [comment]
Related to https://github.com/C9Glax/tranga/issues/168 and implemented in V2
